common/data/content-format-ids.ts

```diff
@@ -1,11 +1,13 @@
 export const ArticleFormatIds = {
-  ImageGallery: 'W5uKaCQAACkA3C0T',
-  // Essay seems to signify an article that is part of a series
-  Essay: 'W7TfJRAAAJ1D0eLK',
+  ImageGallery: 'W5uKaCQAACkA3C0T', // Should this be called In Pictures?  https://wellcomecollection.prismic.io/documents~k=article-formats&b=working&c=published&l=en-gb/W5uKaCQAACkA3C0T/
+  Article: 'W7TfJRAAAJ1D0eLK',
```

Comment:
[Essay was the former name of Article and is not referred to anywhere else anymore,](https://wellcome.slack.com/archives/C3N7J05TK/p1677249901431869) so updating it to avoid further confusion.